l3kernel/l3text.dtx

```diff
@@ -190,11 +192,7 @@
 %       available using \pdfTeX{}.
 %   \end{itemize}
 %
-%  For titlecasing, note that there are two functions available. The
-%  function \cs{text_titlecase:n} applies (broadly) uppercasing to the first
-%  letter of the input, then lowercasing to the remainder. In contrast,
-%  \cs{text_titlecase_first:n} \emph{only} carries out the uppercasing operation,
-%  and leaves the balance of the input unchanged. Determining whether
+%  Determining whether
 %  non-letter characters at the start of text should switch from upper- to
 %  lowercasing is controllable. When \cs{l_text_titlecase_check_letter_bool} is
```

Comment:
I am not sure that the phrase 
"switch from . . . to lower casing"
is meaningful in this context!  
What "switching" is going on with "titlecasing"?

Reply:
I've reworded: this is an area I'd like to go back to anyway as using catcode is not really right, and we now do have Unicode class available - but that's a different PR to this one :)